strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/domain/PypiMetadataDao.java

```diff
@@ -0,0 +1,95 @@
+package org.carlspring.strongbox.domain;
+
+import javax.inject.Qualifier;
+import java.lang.annotation.*;
+
+public class PypiMetadataDao {
```

Comment:
Cosmetic: Would you mind re-indenting all your code according to our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)

Reply:
Done